src/bin/lychee/stats.rs

```diff
@@ -146,11 +149,13 @@ mod test_super {
                     uri: website("http://example.org/failed"),
                     status: Status::Failed(http::StatusCode::BAD_GATEWAY),
                     source: Input::Stdin,
+                    recursion_level: 0,
                 },
                 Response {
                     uri: website("http://example.org/redirect"),
                     status: Status::Redirected(http::StatusCode::PERMANENT_REDIRECT),
                     source: Input::Stdin,
+                    recursion_level: 0,
```

Comment:
For convenience and to cover the "most common case", it might be worth considering adding an alternative constructor for `Response` which would set `recursion_level` to 0.

Or even have `::new` constructor which would have this behavior (recursion = 0) and another one named something like `::with_recursion` which would allow setting recursion level.